client/src/pages/targets/__components__/TargetsListPage/TargetListCardRow.jsx

```diff
@@ -13,7 +13,9 @@ const isChecked = (checkedRadioBtnName, radioBtnName) => {
 const getStatusLabel = (taskStatus) => {
   return (
     (taskStatus === '' || taskStatus === undefined || taskStatus === TARGET_STATUS_NEW) && (
```

Comment:
Could use includes here like taskStatus.includes(' ', undefined, TARGET_STATUS_NEW)

Reply:
That'll fail if `taskStatus` _is_ `undefined`. Assuming it's okay for it to be `null` as well, this should do the trick:
```suggestion
    (!taskStatus || taskStatus === TARGET_STATUS_NEW) && (
```